react/features/base/participants/reducer.js

```diff
@@ -215,10 +224,19 @@ ReducerRegistry.register('features/base/participants', (state = DEFAULT_STATE, a
             };
         }
 
-        state.remote.set(participant.id, participant);
+        state.remote.set(id, participant);
 
-        if (participant.isFakeParticipant) {
-            state.fakeParticipants.set(participant.id, participant);
+        // Insert the new participant.
+        const displayName = name ?? 'Fellow Jitser';
```

Comment:
I that in `getParticipantDisplayName` function we also respect the values in `interfaceConfig.DEFAULT_REMOTE_DISPLAY_NAME` and `interfaceConfig.DEFAULT_LOCAL_DISPLAY_NAME` for display name. Maybe we should match this logic here?

Reply:
Yes I thought about it but saw a FIXME there saying we needed to get rid of the hardcoded string once interfaceConfig gets stored in redux. I will match it for now.

Reply:
Yeah. I see tho that these `interfaceConfig` params are overridable. So I imagine somebody is using them.